src/test/java/frame/FrameTests.java

```diff
@@ -0,0 +1,17 @@
+package frame;
+
+import base.BaseTests;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class FrameTests extends BaseTests {
+
+    @Test
+    public void testFrameText() {
+        var nestedFramesPage = homePage.clickFramesPage().clickNestedFrames();
+        assertEquals(nestedFramesPage.getLeftFrameText(), "LEFT", "Left frame text incorrect");
+        assertEquals(nestedFramesPage.getBottomFrameText(), "BOTTOM", "Bottom frame text incorrect");
```

Comment:
Maybe you should have a diferent test for each, left frame and bottom frame

Reply:
ok